keyboards/ymd67/keymaps/default/readme.md

```diff
@@ -0,0 +1,5 @@
+![YMD67 layout image](https://imgur.com/a/mee1zG4)
+
+# Default YMD67 layout
```

Comment:
This needs to be a direct image link, or the image doesn't show on the page when this file gets rendered.

```suggestion
![YMD67 layout image](https://i.imgur.com/YghD1LH.png)
```